files/nrpe/check_openstack.py

```diff
@@ -1076,6 +1042,7 @@ def parse_command_line():
   parser.add_option("-u", "--username", dest='username', help='username')
   parser.add_option("-p", "--password", dest='password', help='password')
   parser.add_option("-t", "--tenant", dest='tenant', help='tenant name')
+  parser.add_option("-e", "--domain", dest='user_domain_name', help='domain is used for both user_domain_name and project_domain_name, this might need to be updated in the future')
```

Comment:
This seems risky. Is there a technical reason you didn't add user and project domain as separate variables?

Reply:
Addendum: I might be ok with this if the variable is changed to reflect it's current use, e.g. `user_and_project_domain_name`

Reply:
I did it because I'm lazy and I don't like adding flags to nagios checks. In my head this is only required for authentication, so worst thing _in_my_head_ is that something fails because of authentication. I can update the variable name. Note I have used this reasoning for all the tests that requires domains.